gradle.properties

```diff
@@ -6,7 +6,7 @@ grgitPluginVersion = 3.0.0
 ideaExtPluginVersion = 0.5
 kotlinPluginVersion = 1.3.21
 svntoolsPluginVersion = 2.2.1
-versionsPluginVersion = 0.20.0
+versionsPluginVersion = 0.21.0
```

Comment:
Should we use this new feature and set the release channel to RELEASE to not be informed of release candidates?
https://github.com/ben-manes/gradle-versions-plugin/pull/273

Reply:
Done.